### PR TITLE
power-management: Add section.

### PR DESCRIPTION
**Pull request by flexibeast**
Partially addresses #327 and #337.

### CODE REVIEW

File: src/config/power-management.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+The `acpid` service is installed and enabled by default. ACPI events are handled
+by `/etc/acpi/handler.sh`, which uses [zzz(8)](https://man.voidlinux.org/zzz.8)
+for suspend-to-RAM events.
+
+## elogind
+
+The `elogind` service is provided by the `elogind` package. By default,
+`elogind` listens for and processes ACPI events related to lid-switch activation
+and presses of the *power*, *suspend* and *hibernate* keys. This will conflict
```
Comment (ericonr) [inline src/config/power-management.md:17]: "and the ", perhaps?

File: src/config/power-management.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Power Management
+
+Void provides two approaches to power management:
+[acpid(8)](https://man.voidlinux.org/acpid.8) and
+[elogind(8)](https://man.voidlinux.org/elogind.8).
```
Comment (ericonr) [inline src/config/power-management.md:5]: Feels kind of weird that this paragraph mentions acpid and elogind, but not TLP. They are separate categories of applications, too, so perhaps we could have `## Power management` and `## Power saving` top level headers? Terrible names, just thinking about it.

The takeaway is that I think it makes sense for acpid and elogind to be inside a header.

File: src/config/power-management.md (hunk 1 above)
Comment (flexibeast) [reply src/config/power-management.md:17]: Mm, that sounds more awkward to me.

Comment (ericonr) [reply src/config/power-management.md:17]: "events related to [...] the *power* [...] keys" and "events related to [...] presses of the *power* [...] keys

File: src/config/power-management.md (hunk 2 above)
Comment (flexibeast) [reply src/config/power-management.md:5]: Good points - agreed. i'll see what i can come up with.

File: src/config/power-management.md (hunk 1 above)
Comment (flexibeast) [reply src/config/power-management.md:17]: Okay, i'll change the sentence to:

> By default, `elogind` listens for, and processes, ACPI events related to lid-switch activation and the *power*, *suspend* and *hibernate* keys.

Comment (flexibeast) [reply src/config/power-management.md]: PR updated.

File: src/config/power-management.md (hunk 2 above)
Comment (flexibeast) [reply src/config/power-management.md]: PR updated. i've used the heading titles you suggested as at least placeholders - let me know of any titles you think might be better. :-)

File: src/config/session-management.md [hunk 3]
```diff
@@ -30,16 +30,5 @@ also be one of the mechanisms for rootless [Xorg](./graphical-session/xorg.md).
 To make use of its features, install the `elogind` package and make sure the
 [system D-Bus](#d-bus) is enabled. You might need to log out and in again.
 
-### Troubleshooting
-
 If you're having any issues with `elogind`, [enable](./services/index.md) its
 service, as waiting for a D-Bus activation can lead to issues.
```
Comment (ericonr) [inline src/config/session-management.md:34]: Still leave a comment leading back to power management? Like `elogind also does [power management](./power-management.md).`

File: src/SUMMARY.md [hunk 4]
```diff
@@ -30,6 +30,7 @@
       - [AppArmor](./config/security/apparmor.md)
    - [Date and Time](./config/date-time.md)
    - [Kernel](./config/kernel.md)
+   - [Power Management](./config/power-management.md)
```
Comment (ericonr) [inline src/SUMMARY.md:33]: This should become "Power", but then it's hardly descriptive. Is "Power management and saving" too big/bad?

File: src/config/session-management.md (hunk 3 above)
Comment (ericonr) [reply src/config/session-management.md:34]: More importantly, mention that some considerations can be necessary when installing elogind, which I hope leads to people reading about the potential issues.

File: src/SUMMARY.md (hunk 4 above)
Comment (flexibeast) [reply src/SUMMARY.md:33]: i guess i regard "power saving" as an aspect of "power management". Thinking about it some more, my preferred structure for `power-management.md` would be more like:

    # Power Management
    ## acpid
    ## elogind
    ## Power Saving - tlp

WDYT?

File: src/config/session-management.md (hunk 3 above)
Comment (flexibeast) [reply src/config/session-management.md]: i've just pushed some changes; let me know what you think.

File: src/SUMMARY.md (hunk 4 above)
Comment (ericonr) [reply src/SUMMARY.md:33]: That works, I think.

Comment (ericonr) [reply src/SUMMARY.md:33]: If we decide to document more power saving stuff, they can get sublevel headers later.

File: src/config/session-management.md (hunk 3 above)
Comment (ericonr) [reply src/config/session-management.md]: Looks good. Thanks c:

File: src/SUMMARY.md (hunk 4 above)
Comment (flexibeast) [reply src/SUMMARY.md:33]: Thanks, PR updated.

**Pull request merged**